src/main/target/WORMFC/stm32_flash_f405.ld

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
Why you need this file here. The version in the link directory looks to be the same and should be fine?

Reply:
We spoke about potential bootloader option I was looking towards placing stuff in different sectors. But looks like my bootloader is to big and eventually betaflight freaks out and erases itself.... I will have to either forget about this idea or change sth

Reply:
Can be added later if needed. 1Mb of flash is quite some space. Actual BF will only use a part of it?